python/Ganga/GPIDev/Base/Proxy.py

```diff
@@ -229,24 +244,36 @@ def isType(_obj, type_or_seq):
         return isinstance(obj, bare_type_or_seq)
 
 def getName(_obj):
-    """Strip any proxy and then return an objects name"""
+    """Strip any proxy and then return an objects name
+    Args:
+        _obj (object): This may be an instance or a class"""
     obj = stripProxy(_obj)
     returnable = _getName(obj)
     return returnable
 
 def stripProxy(obj):
-    """Removes the proxy if there is one"""
+    """Removes the proxy if there is one
+    Args:
+        obj (object): This may be an instance or a class
+    """
     return getattr(obj, implRef, obj)
 
 
 def addProxy(obj):
-    """Adds a proxy to a GangaObject"""
+    """Adds a proxy to a GangaObject instance or class
+    Args:
+        obj (GangaObject, class): This may be a Ganga object or class which you're wanting to add a proxy to
```

Comment:
OK, it's not actively used this was but it was intended to be when I cleaned up the methods around this a while ago.
I'll drop this down to the other minor features and code cleanup in this PR.

I don't like that we're inconsistent wit the naming of `addProxy` and `getProxyClass` and had coped we could make this simpler so that people really don't have to care about the technical aspects of whats going on here.